addons/mail/wizard/mail_template_preview.py

```diff
@@ -36,7 +36,7 @@ def default_get(self, fields):
     resource_ref = fields.Reference(string='Record', selection='_selection_target_model')
     lang = fields.Selection(_selection_languages, string='Template Preview Language')
     no_record = fields.Boolean('No Record', compute='_compute_no_record')
-    error_msg = fields.Char('Error Message', readonly=True)
+    error_msg = fields.Text('Error Message', readonly=True)
```

Comment:
Would not change this on stable. It would change from Varchar to Text, which should require an upgrade script.

Reply:
it is an no stored field, I think you don't need any script, no ?

Reply:
Default behavior is to store unless it's computed. It's computed on master, but not here.

Although I guess you could reasonably make it computed here since it's a transiant model anyway...

Reply:
You are right, it is stored. I keep Char and update the view.